.changeset/sweet-cars-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": minor
+---
+
```

Comment:
is this relevant?

Reply:
it's the missing changeset from my previous pr that I forgot

Reply:
I could put it in another pr, but I figured it would eb easier this way

Reply:
that's alright